### PR TITLE
feat(samples): Update docs with Go, PHP, Python samples for transactions

### PR DESCRIPTION
**Pull request by phillipgr4vy**

### CODE REVIEW

File: docs/web-checkout/inspect.mdx [hunk 1]
```diff
@@ -111,6 +153,40 @@ bash curl -i -X POST https://api.{gr4vyId}.gr4vy.app/transactions/fe26475d-ec3e-
 
 ```
 
+ </TabItem>
+ <TabItem value ="go">
```
Comment (cbetta) [inline docs/web-checkout/inspect.mdx:157]:
```suggestion
 <TabItem value="go">
```

File: docs/web-checkout/inspect.mdx [hunk 2]
```diff
@@ -111,6 +153,40 @@ bash curl -i -X POST https://api.{gr4vyId}.gr4vy.app/transactions/fe26475d-ec3e-
...
+if err != nil {
+  t.Errorf(err.Error())
+  return;
+}
+t.Log("Capture status: " + *response.Status)
+
+```
+
+ </TabItem>
+ <TabItem value ="php">
```
Comment (cbetta) [inline docs/web-checkout/inspect.mdx:171]:
```suggestion
 <TabItem value="php">
```

File: versioned_docs/version-1.0.0/web-checkout/inspect.mdx [hunk 3]
```diff
@@ -109,6 +151,40 @@ bash curl -i -X POST https://api.{gr4vyId}.gr4vy.app/transactions/fe26475d-ec3e-
 
 ```
 
+ </TabItem>
+ <TabItem value ="go">
```
Comment (cbetta) [inline versioned_docs/version-1.0.0/web-checkout/inspect.mdx:155]:
```suggestion
 <TabItem value="go">
```

File: versioned_docs/version-1.1.0/web-checkout/inspect.mdx [hunk 4]
```diff
@@ -110,6 +151,41 @@ bash curl -i -X POST https://api.{gr4vyId}.gr4vy.app/transactions/fe26475d-ec3e-
...
+if err != nil {
+  t.Errorf(err.Error())
+  return;
+}
+t.Log("Capture status: " + *response.Status)
+
+```
+
+ </TabItem>
+ <TabItem value ="php">
```
Comment (cbetta) [inline versioned_docs/version-1.1.0/web-checkout/inspect.mdx:168]:
```suggestion
 <TabItem value="php">
```

File: versioned_docs/version-1.1.0/web-checkout/inspect.mdx [hunk 5]
```diff
@@ -110,6 +151,41 @@ bash curl -i -X POST https://api.{gr4vyId}.gr4vy.app/transactions/fe26475d-ec3e-
 ```
 
  </TabItem>
+ <TabItem value ="go">
```
Comment (cbetta) [inline versioned_docs/version-1.1.0/web-checkout/inspect.mdx:154]:
```suggestion
 <TabItem value="go">
```

**Review by cbetta — CHANGES_REQUESTED** [review]
I like it. Happy to merge with the requested changes. 

Just 1 question: Is this a good order for the code samples? Node, cURL, Go, PHP, Python? What would be the most used ones? Maybe put those first?

**Review by steve-gr4vy — APPROVED** [review]
Looks good.

**Review by cbetta — APPROVED** [review]

**Pull request merged**